packages/apps/src/SideBar/index.tsx

```diff
@@ -67,6 +68,9 @@ class SideBar extends React.PureComponent<Props, State> {
           onClick={toggleMenu}
         />
         {this.renderModals()}
+        {this.state.modals.network && (
+          <NetworkModal onClose={this.closeModal('network')}/>
```

Comment:
Prefer not passing params through here since it is effectively re-created each render. Just create an explicit `closeNetworkModal` function